lib/client.js

```diff
@@ -77,9 +78,9 @@ import {
  */
 class Client {
   constructor({
-    directory, registryUri, identityUri, token, tenant, enableCache
+    directory, registryUri, identityUri, token, tenant, enableCache, encodeUri
```

Comment:
Just a general query, should we start adding new options on as part of an options object? certainly if we add anymore it will start to get silly.

Reply:
Do you mean as it's own type, rather than a deconstructed object?